spec/requests/filter_parameter_logging_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'Filter parameter logging spec', type: :request do
+  let(:model) { Patient }
+  let(:request_user) { FactoryBot.create(:user) }
+
+  let(:sync_route) { '/api/v3/patients/sync' }
+
+  let(:build_payload) { -> { build_patient_payload(FactoryBot.build(:patient, registration_facility: request_user.facility)) } }
+  let(:build_invalid_payload) { -> { build_invalid_patient_payload } }
+  let(:update_payload) { ->(record) { updated_patient_payload record } }
+
+  def to_response(patient)
+    Api::V3::PatientTransformer.to_nested_response(patient)
+  end
+
+  include_examples 'v3 API sync requests'
```

Comment:
This brings in a bunch of specs we don't care about, but breaking apart the shared example pieces can wait ...